src/goto-cc/gcc_cmdline.h

```diff
@@ -29,7 +29,7 @@ class gcc_cmdlinet:public goto_cc_cmdlinet
 protected:
   typedef std::vector<std::string> argst;
 
-  bool parse_arguments(const argst &args, bool in_spec_file);
+  bool parse_arguments(const argst &, bool in_spec_file);
```

Comment:
Why leave the parameter name blank, instead of making it `args_to_parse` to match the definition?